src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -923,16 +928,30 @@ public class Color : Disposable, ICloneable
             // Do nothing. Just call for load static values.
         }
 
+        internal static System.Runtime.InteropServices.HandleRef getCPtr(Color obj)
+        {
+            return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.SwigCPtr;
+        }
+
+        internal new System.Runtime.InteropServices.HandleRef SwigCPtr
```

Comment:
Add new keyword looks bad to me. We should found some way to remove this new keywords somedays :(